crate/operator/update_user_password.py

```diff
@@ -84,6 +84,12 @@ async def update_user_password(
                 stdout=True,
                 tty=False,
             )
+            if "ALTER OK" in result:
+                logger.info("... success")
+            else:
+                logger.info("... error. %s", result)
+                raise TemporaryError(delay=config.BOOTSTRAP_RETRY_DELAY)
```

Comment:
Won't this get caught in the `except Exception as e` handler below and then not propagated upwards?

Reply:
Very good point!